src/main/java/frc/trigon/robot/subsystems/arm/arm.java

```diff
@@ -0,0 +1,4 @@
+package frc.trigon.robot.subsystems.arm;
+
+public class arm {
+}
```

Comment:
Class names should be CamelCase. See [Java Naming Conventions](https://www.thoughtco.com/using-java-naming-conventions-2034199)